C/CPUInfo/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "CPUInfo"
+version = v"0.0.20201217"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/pytorch/cpuinfo.git", "5916273f79a21551890fd3d56fc5375a78d1598d"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd cpuinfo
+mkdir build
+cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_LIBDIR=$libdir \
+    -DCPUINFO_BUILD_UNIT_TESTS=OFF \
+    -DCPUINFO_BUILD_MOCK_TESTS=OFF \
+    -DCPUINFO_BUILD_BENCHMARKS=OFF \
+    -DCPUINFO_LIBRARY_TYPE=shared \
+    ..
+cmake --build . -- -j $nproc
+make install
+if [[ $target == *-w64-mingw32 ]]; then
+    cp -a libcpuinfo.dll $libdir
```

Comment:
Never use `cp` in this way
```suggestion
    install -Dvm 755 libcpuinfo.dll "${libdir}/libcpuinfo.dll"
```